app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt

```diff
@@ -335,7 +350,7 @@ class UploadTask(
 
     private fun UploadFile.prepareUploadSession(totalChunks: Int): String? {
         val sessionBody = UploadSession.StartSessionBody(
-            conflict = if (replaceOnConflict()) ConflictOption.VERSION else ConflictOption.RENAME,
+            conflict = if (replaceOnConflict()) VERSION else RENAME,
```

Comment:
Put back `ConflictOption.`.
We import everything except the last class name.

That's why I said to _"Import `UploadTask.Companion.`"_.